packages/host/app/templates/host-freestyle.gts

```diff
@@ -48,8 +47,6 @@ class HostFreestyleComponent extends Component<HostFreestyleSignature> {
   }
 
   <template>
-    {{pageTitle 'Host Components'}}
```

Comment:
this collides with our dynamic <title> element, and `pageTitle` complains loudly that there can't 2 title elements.